l10n/en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Question (possibly blocking): do we need to add a config for this file in `pontoon.toml`?

Reply:
Doh, yes, I always forget that part 🤦